Builds/VisualStudio2015/RippleD.vcxproj

```diff
@@ -3353,6 +3353,10 @@
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='debug|x64'">True</ExcludedFromBuild>
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='release|x64'">True</ExcludedFromBuild>
     </ClCompile>
+    <ClCompile Include="..\..\src\ripple\rpc\tests\AccountOffers.test.cpp">

```

Comment:
`git show` shows `^M` control-M line-ends for these 4 lines, and the 3 in the vcxproj.filters file.
Looks like on mac you may need to do `git config --global core.autocrlf input`